.github/workflows/tutorial-quickstarts.yml

```diff
@@ -0,0 +1,56 @@
+name: tutorial-quickstarts
+on:
+  schedule:
+    - cron: "0 0/1 * * *"
+  pull_request:
+    branches:
+      - main
+    paths:
+      - tutorials/quickstarts/**
+jobs:
+  build:
+    runs-on: ubuntu-latest 
+    steps:
+    - name: check out repo
+      uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.6"
+    - name: pip install
+      run: pip install -r requirements.txt 
+    - name: azure login
+      uses: azure/login@v1
+      with:
+        creds: ${{secrets.AZ_AE_CREDS}}
+    - name: install azmlcli
+      run: az extension add -s https://azurecliext.blob.core.windows.net/release/azure_cli_ml-1.15.0-py3-none-any.whl -y
+    - name: attach to workspace
+      run: az ml folder attach -w default -g azureml-examples
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment: azureml_py36
+        environment-file: tutorials/quickstarts/environment.yml
+        python-version: 3.6
+        auto-activate-base: false
+    - name: Conda info
+      shell: bash -l {0}
+      run: conda info
+    - name: Pip install ++
```

Comment:
If you mean the pip installs, they are included in the conda environment since it is the one activated in the shell at the time. But I could add them as part of the environment.yml, the reason I didn't was that I thought it might be an improvement to download the environment file from the Github location where I found it, to make sure it is always up to date.